src/audits/performance/first-meaningful-paint.js

```diff
@@ -61,13 +67,13 @@ class FirstMeaningfulPaint extends Audit {
           // The fundamental Time To fMP metric
           const firstMeaningfulPaint = fmp.firstMeaningfulPaint - fmp.navigationStart;
 
-          // Roughly an exponential curve.
-          //   < 1000ms: penalty=0
-          //   3000ms: penalty=90
-          //   >= 5000ms: penalty=100
-          const power = (firstMeaningfulPaint - 1000) * 0.001 * 0.5;
-          const penalty = power > 0 ? Math.pow(10, power) : 0;
-          let score = 100 - penalty;
+          // Use the CDF of a log-normal distribution for scoring.
+          //   < 1100ms: score≈100
+          //   4000ms: score=50
+          //   >= 14000ms: score≈0
+          const distribution =
+              TracingProcessor.getLogNormalDistribution(SCORING_MEDIAN, SCORING_FALLOFF);
+          let score = 100 * distribution.computeComplementaryPercentile(firstMeaningfulPaint);
 
           // Clamp the score to 0 <= x <= 100.
           score = Math.min(100, score);
```

Comment:
numerical error here is more or less unavoidable, so I think we'll want to clamp. For instance, see the [`erf` function in catapult](https://github.com/catapult-project/catapult/blob/ba2262434b10e553a40e1b14de3ee4b34aee9aaa/tracing/tracing/base/math.html#L76), where some error is accepted because there is no closed form expression of the function. A number of steps are then done on this value to get our CDF, each of which also bring in numerical error, but the error is all quite small in the end.

FWIW, in Ben's original catapult PR where he introduced both the log-normal statistics in there now *and* scoring, he also clamped the scoring coming out of it.